mod/assets/set/abstract/mod_assets.rb

```diff
@@ -130,14 +130,15 @@ def ensure_item_save card, args
   if card
     card.update args
   else
-    Card.create args.merge(name: item_name)
+    Card.create! args
```

Comment:
Not sure where this "item_name" was coming from, but I don't think it's what we wanted.